pages/api/v1/saml/config/exists.ts

```diff
@@ -0,0 +1,30 @@
+import jackson from '@lib/jackson';
+import { extractAuthToken, validateApiKey } from '@lib/utils';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const apiKey = extractAuthToken(req);
+    if (!validateApiKey(apiKey)) {
+      res.status(401).json({ message: 'Unauthorized' });
+      return;
+    }
+
+    const { apiController } = await jackson();
+    if (req.method === 'GET') {
+      const rsp = await apiController.getConfig(req.query as any);
```

Comment:
Just to confirm, does this line mean that the exposed interface for the `api/v1/saml/config/exists` accept the same inputs as the `api/v1/saml/config`:

> You can use either clientID or tenant and product combination.

Reply:
@nadilas Indeed, same inputs as the `api/v1/saml/config` API